test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
This file needs to be ignored with .gitignore. Since you already uploaded to Github, even if you ignore it, it will still exist there. Please so delete it from Github directly.